CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * Improve broker scale-down prevention to continue in reconciliation when scale-down cannot be executed
 * Added support for Tiered Storage by enabling the configuration of custom storage plugins through the Kafka custom resource.
 * Update HTTP bridge to latest 0.28.0 release
+* Added support for topic replication factor change to the Unidirectional Topic Operator when Cruise Control integration is enabled.
```

Comment:
Could you please move it to the 0.41.0 section? Then we should be able to merge it. Thanks.